dlrs/clearlinux/tensorflow/oss/README.md

```diff
@@ -10,7 +9,7 @@ Default build args in Docker are on: https://docs.docker.com/engine/reference/bu
 >NOTE: This command is for locally building this image alone.
 
 ```
-docker build --no-cache --build-arg clear_ver="31290" -t clearlinux/stacks-dlrs-oss .
+docker build --no-cache --build-arg clear_ver="30650" -t clearlinux/stacks-tensorflow-oss .
```

Comment:
clear_ver should be 32690, also please change the tag to dlrs-tensorflow-clearlinux:v0.6.0